code/modules/status_system/statusEffects.dm

```diff
@@ -1277,7 +1277,7 @@
 		maxDuration = null
 		effect_quality = STATUS_QUALITY_POSITIVE
 		var/const/max_health = 30
-		var/const/max_stam = 60
+		var/const/max_stam = 20
```

Comment:
Actually thinking about it, this is pretty low now and we still want people to be incentivized to wear their uniforms. Maybe make it ~40 instead so it juust doesn't cross the threshold for baton hits?

Reply:
sure. why not